content/intro-to-storybook/angular/en/simple-component.md

```diff
@@ -279,99 +258,57 @@ Make sure your components render data that doesn't change, so that your snapshot
 With the [Storyshots addon](https://github.com/storybooks/storybook/tree/master/addons/storyshots) a snapshot test is created for each of the stories. Use it by adding a development dependency on the package:
 
 ```bash
-yarn add -D @storybook/addon-storyshots identity-obj-proxy jest jest-preset-angular
-```
-
-Then create an `src/storybook.test.ts` file with the following in it:
-
-```typescript
-import * as path from 'path';
-import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
-
-initStoryshots({
-  framework: 'angular',
-  configPath: path.join(__dirname, '../.storybook'),
-  test: multiSnapshotWithOptions(),
-});
+npm install -D @storybook/addon-storyshots babel-plugin-require-context-hook
 ```
 
-After that, create a `src/jest-config` folder with two files inside, `globalMocks.ts`:
+Update the `babel.config.js` that was created in the previous section to the following:
 
-```typescript
-const mock = () => {
-  let storage = {};
+```javascript
+module.exports = function(api) {
+  const isTest = api.env('test');
+  const presets = [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+    '@babel/preset-typescript',
+  ];
```

Comment:
According to the review on the 'Get Started' section, it might be required to add '@babel/preset-react' too

Reply:
@geromegrignon the `@babel/preset-react` will only be required in the end of the tutorial. When the user is introduced on how to create a simple addon.

Reply:
@geromegrignon i'm marking this as resolved if you don't mind